src/Tokenomia/Vesting/PrivateSale.hs

```diff
@@ -0,0 +1,235 @@
+{-# LANGUAGE BlockArguments #-}
```

Comment:
A lot (most?) of these pragmas are redundant. Please remove the unnecessary ones. Preferably pragmas should go into the cabal file, unless it's _really_ something you don't want to enable for all modules